### PR TITLE
Remove Jasmine

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

**Comment by gzm0** [comment]
Review by @sj

**Comment by gzm0** [comment]
Manually tested HTML test on 2.11 in fast and full opt of `testingExample`.

File: examples/testing/src/test/scala/ElementCreatorTest.scala [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-      val body = global.document.getElementsByTagName("body")
-        .asInstanceOf[js.Array[js.Dynamic]].head
-
-      // the Scala.js DOM API would make this easier
-      expect(body.lastChild.tagName.toString == "H1").toBeTruthy
-      expect(body.lastChild.innerHTML.toString == "Test").toBeTruthy
-    }
+    // the Scala.js DOM API would make this easier
+    assertEquals(body.lastChild.tagName.toString, "H1")
+    assertEquals(body.lastChild.innerHTML.toString, "Test")
```
Comment (nicolasstucki) [inline examples/testing/src/test/scala/ElementCreatorTest.scala]: The parameters should be inverted. `def assertEquals(expected: AnyRef, actual: AnyRef)`

**Comment by nicolasstucki** [comment]
That is all.

**Comment by gzm0** [comment]
Updated.

**Comment by nicolasstucki** [comment]
LGTM

File: project/Build.scala [hunk 2]
```diff
@@ -1189,7 +1171,8 @@ object Build {
             "org.webjars" % "jquery" % "1.10.2" / "jquery.js" % "test"
           )
       )
-  ).withScalaJSCompiler.dependsOn(library, jasmineTestFramework % "test")
+  ).withScalaJSCompiler.withScalaJSJUnitPlugin
+      .dependsOn(library, jUnitRuntime % "test")
```
Comment (gzm0) [inline project/Build.scala]: Just to be sure: Is this according to the style guide?

Comment (sjrd) [inline project/Build.scala]: The style guide is not really precise for the sbt DSL. But you can see at `jUnitTestOutputsJS` that it should look like this:
```scala
  ).withScalaJSCompiler.withScalaJSJUnitPlugin.dependsOn(
      library, jUnitRuntime % "test"
  )
```

**Comment by sjrd** [comment]
That's all.

**Comment by gzm0** [comment]
Updated

**Comment by sjrd** [comment]
LGTM

**Pull request merged**